tests/unit/domain/sing/shouldPlayTracks.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { createDefaultTrack, shouldPlayTracks } from "@/sing/domain";
+import { Track } from "@/store/type";
+import { TrackId } from "@/type/preload";
+
+const createTrack = ({ solo, mute }: { solo: boolean; mute: boolean }) => {
+  const track = createDefaultTrack();
+
+  if (solo) {
+    track.solo = true;
+  }
+  if (mute) {
+    track.mute = true;
+  }
+
+  return track;
```

Comment:
（細かいけど）意図としてはこっちのが自然かも
```suggestion
  const track = createDefaultTrack();
  track.solo = solo;
  track.mute = mute;
  return track;
```